### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by loekvangool**

### CODE REVIEW

File: docs/index-operations.asciidoc [hunk 1]
```diff
@@ -240,16 +240,16 @@ The Get Mappings API will return the mapping details about your indexes and type
 $response = $client->indices()->getMapping();
 
 // Get mappings for all types in 'my_index'
-$params = ['index' => 'my_index'];
+$params = [ 'index' => 'my_index'];
 $response = $client->indices()->getMapping($params);
 
 // Get mappings for all types of 'my_type', regardless of index
-$params = ['type' => 'my_type' ];
+$params = [ 'type' => 'my_type' ];
```
Comment (Oliboy50) [inline docs/index-operations.asciidoc]: you could also remove the last space after `my_type` then

Comment (loekvangool) [inline docs/index-operations.asciidoc]: That would not make the file consistent yet. Do you know the convention? I will update the file to match it.

Comment (polyfractal) [inline docs/index-operations.asciidoc]: I don't think there is a convention, but I'd prefer it to at least be consistent within the page.  E.g. either spaces before/after, or no spaces at all.

I think I'd lean towards no spaces before/after personally.

**Comment by ezimuel** [comment]
Closing due to inactivity.

**Pull request closed without merging**